src/store/flow/views/import.ts

```diff
@@ -146,6 +146,7 @@ export const actions: ActionTree<IImportState, IRootState> = {
     try {
       // check valid json
       flowContainer = JSON.parse(state.flowJsonText)
+      commit('setFlowJsonText', JSON.stringify(flowContainer, null, 2))
```

Comment:
Once we know it's valid json we can set it formatted immediately. We then perform other validation steps.

Reply:
@jcbashdown , for readability I'd prefer we put this after the `try catch`. Besides, that will avoid us confusion in case the error comes from `setFlowJsonText( )` or `JSON.stringify(flowContainer, null, 2)` (for now it's simple, but we may alter this mutation in the future)

What do you think?
I'm approving and let you decide which is the best.

Reply:
Good idea. I'll move it.